pkg/controller/agent/config_test.go

```diff
@@ -64,8 +78,22 @@ func TestBuildFleetSetupKibanaConfig(t *testing.T) {
 			client:  client,
 		},
 		{
-			name:  "kibana ref present",
-			agent: *assoc.Agent,
+			name:  "kibana ref present, kibana with ca populated",
+			agent: *assocWithoutCa.Agent,
```

Comment:
It's the other way round this one is the one withou CA and the one on L95 is with CA.